src/main/java/com/laytonsmith/abstraction/bukkit/BukkitConvertor.java

```diff
@@ -476,6 +477,9 @@ public MCInventory GetLocationInventory(MCLocation location) {
 
 	@Override
 	public void runOnMainThreadLater(DaemonManager dm, final Runnable r) {
+		if(!CommandHelperPlugin.self.isEnabled()) {
+			throw new CancelCommandException("CommandHelper tried to schedule a task while the plugin was disabled (is the server shutting down?).", Target.UNKNOWN);
```

Comment:
Use Implementation.GetServerType().getBranding() instead of hard coding the string "CommandHelper". This applies in both places.